apps/web/composables/useOrderAgain/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { Order } from '@plentymarkets/shop-api';
```

Comment:
I would double-check & refactor all these ```type``` imports, as some of them are not accurate, leading to unexpected results (eg: here we are dealing with an interface)

```import { type Xxxxx, Yyyy };``` is more appropiate imo